Examples/LocationManager/LocationManager/LocalSearchClient/LocalSearchModels.swift

```diff
@@ -0,0 +1,202 @@
+import MapKit
+
+struct LocalSearchResponse: Equatable {
+  var mapItems: [MapItem]
+  var boundingRegion: MKCoordinateRegion
+
+  init(
+    response: MKLocalSearch.Response
+  ) {
+    self.mapItems = response.mapItems.map(MapItem.init(rawValue:))
+    self.boundingRegion = response.boundingRegion
+  }
+
+  init(
+    mapItems: [MapItem],
+    boundingRegion: MKCoordinateRegion
+  ) {
+    self.mapItems = mapItems
+    self.boundingRegion = boundingRegion
+  }
+
+  static func == (lhs: Self, rhs: Self) -> Bool {
+    lhs.mapItems == rhs.mapItems
+      && lhs.boundingRegion.center.latitude == rhs.boundingRegion.center.latitude
+      && lhs.boundingRegion.center.longitude == rhs.boundingRegion.center.longitude
+      && lhs.boundingRegion.span.latitudeDelta == rhs.boundingRegion.span.latitudeDelta
+      && lhs.boundingRegion.span.longitudeDelta == rhs.boundingRegion.span.longitudeDelta
+  }
+}
+
+struct MapItem: Equatable {
```

Comment:
same here, `MapItem` is not equatable or constructible, so we need a testing friendly version of it.